modules/oxide/src/less/theme/components/menubar/promotion.less

```diff
@@ -0,0 +1,34 @@
+//
+// Promotion
+//
+
+@promotion-background-color: @background-color;
+@promotion-background: url("data:image/svg+xml;charset=utf8,%3Csvg height='@{_promotion-height}' viewBox='0 0 40 @{_promotion-height}' width='40' xmlns='http://www.w3.org/2000/svg'%3E%3Crect x='0' y='@{_promotion-divider-height}' width='100' height='1' fill='@{_promotion-row-separator-color}'/%3E%3C/svg%3E") left 0 top 0 @promotion-background-color;
+
+// private variables
+@_promotion-row-separator-color: escape('@{menubar-row-separator-color}');
+@_promotion-height: @_promotion-select-height + (@_promotion-select-spacing-y * 2) + 1px;
+@_promotion-divider-height: @_promotion-height - 1px;
+@_promotion-select-height: @menubar-select-height;
+@_promotion-select-spacing-y: @menubar-select-spacing-y;
+
+.tox {
+  .tox-promotion {
+    background: @promotion-background;
+    background-color: @promotion-background-color;
+    cursor: pointer;
+    grid-column: 2;
+    grid-row: 1;
+    padding-inline-end: 8px;
+    padding-inline-start: 4px;
+    padding-top: 5px;
+  }
+
+  .tox-promotion-link {
```

Comment:
I noticed while reviewing the design spec today this is missing the hover states, so that needs adding/fixing as well. According to the figma details it's supposed to have the following background colors:
- Normal - #E8F1F8
- Hover - #B4D7FF

The design doc also had an icon next to the text... not sure if that's required but it'd be worth double checking or maybe putting in the follow up Jira.

- [ ] Fix hover/focus styles to make the design document